static-analysis/teammates-eslint-json.yml

```diff
@@ -1,629 +1,5 @@
 env:
     browser: true
-    jquery: true
-    qunit: true
 
 plugins:
     - json
```

Comment:
You should explain the reason for this in your commit message. Otherwise someone from the future looking at this will have no idea why you're keeping JSON linting or renaming the file to `teammates-eslint-json.yml`.